src/services/api.ts

```diff
@@ -64,6 +64,7 @@ export let postProviders = (providers: WriteProvidersRequest[]) =>
 
 type ReadRepresentativePointsResponse = {
   county: string
+  census_tract?: string
```

Comment:
Why is this optional? Try to make it required or remove it.

Reply:
It does not exist yet in the dataset but will be passed as null by the backend.

Reply:
2 options:

1. Remove it from here until we actually add it
2. Type it as null: 'census_tract: null'